clients/java/pom.xml

```diff
@@ -136,6 +136,10 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.zeebe</groupId>
+      <artifactId>zeebe-util</artifactId>
```

Comment:
Hm, I'm not sure about adding a new dependency to the client, esp. since the client __has__ to be Java 8 only. Why don't we just treat the keep alive as we do the other durations in the client? e.g. the `requestTimeout`, or the `pollInterval`?

Reply:
It was just because it's easier for the user to reason about 1.5m than 90000 ms. But you're right, I'll change the clients to that

Reply:
I agree, but I prefer that (+ being consistent) to adding a dependency to the client module. Thanks!